### PR TITLE
4829 display list of judges in advanced order search UI (for both public and authorized user ui)

### PR DESCRIPTION
**Pull request by mrinsin**
image

Also in this PR: including all documents with order event codes in the search filter

### CODE REVIEW

File: docs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -102,4 +102,15 @@ After this successfully runs, you can navigate to the project root (via `cd ../.
 
 - `./web-api/run-serverless-clamav.sh <env> <region>`, where `<env>` would be your environment (dev, stg, prod, etc.) and `<region>` would be your AWS region (us-east-1, us-west-1, etc.)
 
-To revert your `serverless-prune-plugin`, just change `before` back to `after` or run `npm i` again.
\ No newline at end of file
+To revert your `serverless-prune-plugin`, just change `before` back to `after` or run `npm i` again.
```
Comment (mrinsin) [inline docs/TROUBLESHOOTING.md]: Not sure how this got into my PR, I only pulled develop, but this file is not a part of the changes that were made for this PR

File: shared/src/business/useCases/processStreamRecordsInteractor.js [hunk 2]
```diff
@@ -1,6 +1,57 @@
 const AWS = require('aws-sdk');
 const { createISODateString } = require('../utilities/DateHandler');
 
+/**
```
Comment (mrinsin) [inline shared/src/business/useCases/processStreamRecordsInteractor.js]: Not sure how this got into my PR, I only pulled develop, but this file is not a part of the changes that were made for this PR

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/useCases/users/getUsersInSectionInteractor.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ exports.getUsersInSectionInteractor = async ({
   section,
 }) => {
   const user = applicationContext.getCurrentUser();
-  if (!isAuthorized(user, ROLE_PERMISSIONS.GET_USERS_IN_SECTION)) {
+  if (!isAuthorized(user, ROLE_PERMISSIONS.GET_JUDGES)) {
```
Comment (rachaelparris) [inline shared/src/business/useCases/users/getUsersInSectionInteractor.js]: You may still be working on this, but you'll want to check if the section is `judge` before checking this permission. If it's not judge, then check the `GET_USERS_IN_SECTION` permission.

Comment (mrinsin) [reply shared/src/business/useCases/users/getUsersInSectionInteractor.js]: Updating it right now - we were a little confused about that and were hoping someone would make a suggestion in the PR, so thank you!!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**